src/api/v4/filter/bounding-box.js

```diff
@@ -0,0 +1,72 @@
+var _ = require('underscore');
+var Base = require('./base');
+var BoundingBoxFilterModel = require('../../../windshaft/filters/bounding-box');
+
+/**
+ * Bounding box filter
+ *
+ * @constructor
+ * @extends carto.filter.Base
+ * @memberof carto.filter
+ * @api
+ *
+ */
+function BoundingBox () {
+  this._internalModel = new BoundingBoxFilterModel();
+}
+
+BoundingBox.prototype = Object.create(Base.prototype);
+
+/**
+ * Set the bounds
+ *
+ * @param  {carto.filter.Bounds} bounds
```

Comment:
Is there JSDoc for `carto.filter.Bounds`?

Reply:
Good point!